lua/Network.lua

```diff
@@ -220,25 +220,71 @@ local function isNetworkConnected(linesConnected)
 	end)
 
 	if getn(linesConnected) > 1 then
-		local first
-		local second
-
-		forEachOrderedElement(linesConnected, function(idx)
-			if not first then
-				first = idx
-			elseif not second then
-				second = idx
-			else
-				return false
-			end
-		end)
-
-		return false, first, second
+		return false
 	end
 
 	return true
 end
 
+local function addNetIdInfo(netIdName, cs, linesConnected)
+	local netId = 0
+	for _, v in pairs(linesConnected) do
+		for i = 1, #v do
+			forEachCell(cs, function(cell)
+				if not cell[netIdName] then
+					if cell.FID == v[i] then
+						cell[netIdName] = netId
+					end
+				end
+			end)
+		end
+		netId = netId + 1
+	end
+end
+
+local function saveErrorInfo(self, linesConnected)
+	local gis = getPackage("gis")
+	local linesCs = self.linesCs
+	local errorLayerName = "neterror"
+	local netIdName = "net_id"
+	local errMsg
+
+	if linesCs.project then
+		addNetIdInfo(netIdName, linesCs, linesConnected)
+		linesCs:save(errorLayerName, netIdName)
+		errMsg = "It was created a new Layer '"..errorLayerName
```

Comment:
Layer ... was automatically created with attribute ... containing the separated networks.